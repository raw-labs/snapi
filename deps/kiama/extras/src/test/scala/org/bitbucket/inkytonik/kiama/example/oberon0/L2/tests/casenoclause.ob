MODULE casenoclause;

VAR
    avar : INTEGER;

BEGIN
    CASE avar OF
    END
END casenoclause.
