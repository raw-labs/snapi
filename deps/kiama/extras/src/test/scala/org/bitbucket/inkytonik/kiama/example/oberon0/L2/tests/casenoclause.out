* errors
src/test/scala/org/bitbucket/inkytonik/kiama/example/oberon0/L2/tests/casenoclause.ob:8:5:error: clause expected
    END
    ^
